### ISSUE TITLE
[studio-ui] Request Publish Dialog should allow submitter to specify environment 

### ISSUE THREAD

**Issue by russdanner** [craftercms/craftercms#2203]
### Expected behavior
Content author should be able to specify what environment they are requesting a publish for via the request publish dialog

### Actual behavior
Today the dialog does not give the author the choice.  Live environment is assumed.

### Steps to reproduce the problem
N/a

### Log/stack trace (use https://gist.github.com)
N/a

#### Version
Studio Version Number: 3.0.14-SNAPSHOT-bb5185
Build Number: bb518515b875d7784bb698317a7d0d3f36e2cbe0
Build Date/Time: 06-08-2018 09:50:34 -0400

#### Browser
N/a

**Comment by vestrada** [comment]
PR: https://github.com/craftercms/studio-ui/pull/864

**Comment by vestrada** [comment]
Hi Dejan,

I'm getting a DB error on Submit-to-go-live api.

`[ERROR] 2018-07-03 17:31:51,453 [http-nio-8080-exec-8] [workflow.WorkflowServiceImpl] | Error while submitting content for approval. 
org.springframework.jdbc.BadSqlGrammarException: 
### Error updating database.  Cause: java.sql.SQLSyntaxErrorException: (conn:10) Unknown column 'submittedtoenvironment' in 'field list'
Query is: UPDATE item_metadata
         SET submittedby=?,
            submittedfordeletion=?,
            sendemail=?,
            submissioncomment=?,
            
            
            submittedtoenvironment=? 
        WHERE site=?
        AND path=?, parameters ['author',0,1,'','live','borr','/site/website/articles/2016/7/new-acme-phone-released-today/index.xml']
### The error may involve org.craftercms.studio.api.v1.dal.ItemMetadataMapper.setProperties-Inline
### The error occurred while setting parameters
### SQL: UPDATE item_metadata          SET submittedby=?,             submittedfordeletion=?,             sendemail=?,             submissioncomment=?,                                       submittedtoenvironment=?          WHERE site=?         AND path=?
### Cause: java.sql.SQLSyntaxErrorException: (conn:10) Unknown column 'submittedtoenvironment' in 'field list'
Query is: UPDATE item_metadata
         SET submittedby=?,
            submittedfordeletion=?,
            sendemail=?,
            submissioncomment=?,
            
            
            submittedtoenvironment=? 
        WHERE site=?
        AND path=?, parameters ['author',0,1,'','live','borr','/site/website/articles/2016/7/new-acme-phone-released-today/index.xml']
; bad SQL grammar []; nested exception is java.sql.SQLSyntaxErrorException: (conn:10) Unknown column 'submittedtoenvironment' in 'field list'
Query is: UPDATE item_metadata
         SET submittedby=?,
            submittedfordeletion=?,
            sendemail=?,
            submissioncomment=?,
            
            
            submittedtoenvironment=? 
        WHERE site=?
        AND path=?, parameters ['author',0,1,'','live','borr','/site/website/articles/2016/7/new-acme-phone-released-today/index.xml']`

**Comment by dejan-brkic** [comment]
Vero, your database is not up to date. It is missing the new column.
Request publish dialog works fine. API updates database correctly. Items waiting for the approval dashboard displays correctly selected environment. Approve dialog does not prepopulate selected environment

**Comment by vestrada** [comment]
Hi Dejan,

It is not working when you select a specific date for the "request publish".

**Comment by dejan-brkic** [comment]
Fixed:
https://github.com/craftercms/studio/pull/1176
